index.html

```diff
@@ -856,6 +856,22 @@ <h3>
           </li>
         </ol>
       </li>
+      <li>If the session's [=next wallet behavior=] is set:
+        <ol>
+          <li>If the mode is "respond", resolve the promise with a new
```

Comment:
Nit: let's link these up properly to their definitions (e.g. {{DigitalCredential}})... "respond" should be formally defined, etc.